packages/ui/src/Select/Combobox.tsx

```diff
@@ -357,12 +391,33 @@ export function Combobox(props: ComboboxProps) {
     onChange: props.onValueChange,
   })
 
+  // Get all available options from children
+  const options = React.useMemo(() => {
+    const options: string[] = []
+    React.Children.forEach(children, (child) => {
+      if (React.isValidElement(child) && child.props.value) {
+        options.push(child.props.value)
+      }
+    })
+    return options
+  }, [children])
+
   const selectValue = (val) => {
     if (!value) return
-    if (value.includes(val)) {
-      setValue(value!.filter((v) => v !== val))
+    if (val === 'select-all') {
```

Comment:
The `select-all` might be overridden by the user's `<SelectItem />`, so we better replace it with a more unique value like `$$SELECT_ALL$$` or a Symbol value.

And IMO most of the time this feature is really needed when we have a lot of selected items. Adding a `prop` to let people enable/disable this feature would be good.

Reply:
Or just don't provide this feature at this time; instead, let the user implement it by adding a selection `<SelectItem value="select-all" />` and using `value` and `onValueChange` to control the select-all feature.

Reply:
Done. Replace `select-all` with`$$SELECT_ALL$$`, and add a `showSelectAll` prop (default value is `false`) to let people enable/disable this feature.